### ISSUE TITLE
--device and --connection parameters get ignored

### ISSUE THREAD

**Issue by trieloff** [calibreapp/cli#5]
## CLI Installation method

* [x] NPM
* [ ] Binary

## Operating system

* [ ] Mac
* [ ] Windows
* [x] Linux

## What you expected to happen

I've been running this command:

```bash
$ ./node_modules/.bin/calibre test create http://$CIRCLE_BRANCH.$MYURL --location=SaoPaulo --device=iPhone8 --connection=good3G
```

According to the documentation, specifying *iPhone8* and *good3G* as the device and connection parameters.

I expected a test to be run with these conditions.

## What happened

```bash
http://$CIRCLE_BRANCH.$MYURL on Chrome Desktop
```

The `--device` and `--connection` parameters have been ignored without any error message.

## How to improve

either:
- add a `device-list` and `connection-list` subcommand similar to `location-list`
- document the correct parameters and their possible values
- report an error for unknown devices and connection properties
- all of the above

**Comment by benschwarz** [comment]
Hey @trieloff ! 

That is super strange, I just tried the command and it appears to have created the run correctly: 

```
➜ calibre test create https://calibreapp.com --location=SaoPaulo --device=iPhone8 --connection=good3G
✔ Test created: e3c2ddf
```

When I list all tests, I can see it: 

`e3c2ddf | calibreapp.com/      | iPhone 8         | Not Throttled | 🇧🇷  São Paulo      | Completed 6:35pm 8-Mar-2018`

If I create another test that drops the 'g' off 'good3g': 

```
➜ calibre test create https://calibreapp.com --location=SaoPaulo --device=iPhone8 --connection=good3
✖ Expected "good3" to be one of: regular2G, good2G, slow3G, regular3G, good3G, emergingMarkets, regular4G, dsl, wifi
```
Could you double check the command that you used? I ran my tests on a linux host to make sure it's not a platform dependent issue so I'm a bit lost for ideas at this point.

**Comment by trieloff** [comment]
It looks like there is a discrepancy between the `stdout` of `test create` and of `test list`

Re-Running the test still reports "Chrome Desktop" in the `stdout` of `test create`, but the `test list` correctly reports it to be running on an iPhone.

But in both cases, the `--connection` parameter does not seem to affect the output of `test list`, the "connection" column shows "Not Throttled" for all tests.

**Comment by benschwarz** [comment]
Closed by #3ad2f33c6438eac975c0dbd97122ff92530dadeb
